state/modeluser.go

```diff
@@ -193,6 +193,19 @@ func (e *UserModel) LastConnection() (time.Time, error) {
 	return lastConnDoc.LastConnection, nil
 }
 
+// AllUserModels returns a list of all models with the user they belong to.
+func (st *State) AllUserModels() ([]*UserModel, error) {
+	models, err := st.AllModels()
+	if err != nil {
+		return nil, errors.Trace(err)
+	}
+	var result []*UserModel
+	for _, model := range models {
+		result = append(result, &UserModel{Model: model, User: model.Owner()})
```

Comment:
The idea is to abstract the caller from building these UserModel structs, it was quite noisy to me that we can only ask these for one particular user.
But if I dont manage to convince you that is a good sign that I might be wrong.